docs/organizations/managing-people.md

```diff
@@ -1,17 +1,19 @@
 # Managing people
 
-Members of a Codacy organization can access the Codacy app to see the dashboards and details of the organization repositories. Depending on their permissions on the Git provider, members can also manage the organization and repository settings on Codacy.
+People signing up to the Codacy app and joining an organization become members of that organization. Members of a Codacy organization can see the dashboards and details of the organization repositories. Depending on their permissions on the Git provider, members can also manage the organization and repository settings on Codacy.
 
-For **private repositories**, Codacy only analyzes commits from members of the corresponding organization on Codacy.
+People committing to private repositories of an organization but haven't signed up to Codacy and joined the organization yet, should also be added to the organization.
+
+For **private repositories**, Codacy only analyzes commits from members and committers of the corresponding organization on Codacy .
```

Comment:
This part should be more straightforward, focusing on explaining in a simple way what users should do.

Here's my try:

```suggestion
For **private repositories**, Codacy only analyzes commits from people listed under the page **People** under the organization settings. To make sure that Codacy analyzes all relevant commits, make sure that you add all committers to private repositories to your Codacy organization.
```

Reply:
I wouldn't mention directly the page **People** because we would be repeating the same right after, on the next sentence after the admonition.
I tweaked your suggestion a bit. I removed the "committers **to private repositories**" to simplify the sentence, as we can infer by the beginning of the sentence that we refer to private repositories.
I also removed the duplicated "make sure". 😉 

WDYT, @prcr? 

```suggestion
For **private repositories**, Codacy only analyzes commits from people in your Codacy organization. To ensure that Codacy analyzes all relevant commits, make sure that you add to your Codacy organization the committers that are not members of the organization yet.
```

Reply:
Let's remove the "make sure that you" part (to avoid the "to ensure [...] make sure" :smile:) and it's perfect! :ok_hand: